Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -258,6 +258,30 @@ extension BrowserViewController: WKNavigationDelegate {
                 self.tabManager.selectedTab?.alertShownCount = 0
                 self.tabManager.selectedTab?.blockAllAlerts = false
             }
+            
+            // Prevents synthetically activated links such as: CVE-2017-7089
+            if ["data", "blob", "file"].contains(url.scheme) {
+                if let clickType = navigationAction.value(forKey: "syntheticClickType") as? Int {
+                    if clickType == 0 {
```

Comment:
how do we know this returns an int and the value is `0`?
Is there any documentation for it? If so you could add some comment in code